### PR TITLE
RMMIS-6507: Enable ControlledTabs/ControlledTab

### PR DESCRIPTION
**Pull request by traviscooperscdhhs**
Moved ControlledTabs (container) and ControlledTab (tabbed content) to UIC.  Updated Field for single checkboxes and radios to properly render from config/props. Updated react bootstrap to latest version to take advantage of new Tabs/Tab components

### CODE REVIEW

File: src/ControlledTabs.jsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    this.handleSelect = this.handleSelect.bind(this);
+  }
+
+  handleSelect(key) {
+    let _div = React.findDOMNode(this);
+    let event = new Event('change', {bubbles: true, cancelable: true});
+    event.component = {
+      id: this.props.id,
+      schemaUpdates: {
+        components: {
```
Comment (mallenscdhhs) [inline src/ControlledTabs.jsx]: See @kjschulz 's PR for Hours of Operation, he updated the `_handlePageChange` method to format the schemaUpdates properly

File: src/index.js [hunk 2]
```diff
@@ -5,6 +5,8 @@ module.exports = {
 	alert: require('react-bootstrap').Alert,
 	content: require('./Content'),
 	contentEditor: require('./ContentEditor'),
+	controlledtab: require('./ControlledTab'),
```
Comment (mallenscdhhs) [inline src/index.js]: can we just call these "tab" and "tabs"?

Comment (kjschulz) [inline src/index.js]: :+1:

File: src/ControlledTabs.jsx (hunk 1 above)
Comment (traviscooperscdhhs) [inline src/ControlledTabs.jsx]: updated! (gitrdone)

File: src/index.js (hunk 2 above)
Comment (traviscooperscdhhs) [inline src/index.js]: Updated schemaUpdate format to correct format.

Comment (mallenscdhhs) [inline src/index.js]: @traviscooper can you update the names of these components to `tab` and `tabs`?

Comment (traviscooperscdhhs) [inline src/index.js]: will do!

**Comment by traviscooperscdhhs** [comment]
updated PR with new 'tabs' and 'tab' names.

**Pull request merged**